### PR TITLE
Shady nagy/videos

### PR DESCRIPTION
**Pull request by ShadyNagy**

### CODE REVIEW

File: tests/DevBetterWeb.Vimeo.Tests/Helpers/HttpServiceBuilder.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  {
+    public static HttpService Build()
+    {
+      var httpClient = new HttpClient { BaseAddress = new Uri(ServiceConstants.VIMEO_URI) };
+      httpClient.DefaultRequestHeaders.Remove("Accept");
+      httpClient.DefaultRequestHeaders.Add("Accept", ServiceConstants.VIMEO_HTTP_ACCEPT);
+      httpClient.Timeout = TimeSpan.FromMinutes(60);
+      var httpService = new HttpService(httpClient);
+
+      var vimeoToken = Environment.GetEnvironmentVariable("VIMEO_TOKEN");
```
Comment (ardalis) [inline tests/DevBetterWeb.Vimeo.Tests/Helpers/HttpServiceBuilder.cs:18]: Replace magic string with constant. Make it clear the constant is the name of the config section that holds the actual key.

**Review by ardalis — APPROVED** [review]

File: tests/DevBetterWeb.Vimeo.Tests/Helpers/HttpServiceBuilder.cs [hunk 2]
```diff
@@ -16,7 +16,8 @@ public static HttpService Build()
       httpClient.Timeout = TimeSpan.FromMinutes(60);
       var httpService = new HttpService(httpClient);
 
-      httpService.SetAuthorization(ConfigurationConstants.VIMEO_TOKEN);
+      var vimeoToken = Environment.GetEnvironmentVariable(ConfigurationConstants.VIMEO_TOKEN);
+      httpService.SetAuthorization(vimeoToken);
```
Comment (ardalis) [inline tests/DevBetterWeb.Vimeo.Tests/Helpers/HttpServiceBuilder.cs:20]: 👍

Comment (ShadyNagy) [reply tests/DevBetterWeb.Vimeo.Tests/Helpers/HttpServiceBuilder.cs:20]: done

File: tests/DevBetterWeb.Vimeo.Tests/Helpers/HttpServiceBuilder.cs (hunk 1 above)
Comment (ShadyNagy) [reply tests/DevBetterWeb.Vimeo.Tests/Helpers/HttpServiceBuilder.cs]: done

**Comment by ardalis** [comment]
Going to approve but we still need to deal with the token

**Pull request merged**